### PR TITLE
Add new midstream branches and update pipelines

### PR DESCRIPTION
**Pull request by serverless-qe**
Discover new branches and update Konflux pipelines using openshift-knative/hack.

### CODE REVIEW

File: config/client.yaml [hunk 1]
```diff
@@ -3,9 +3,6 @@ config:
     release-v1.15:
       konflux:
         enabled: true
-      imageOverrides:
-      - name: CLI_ARTIFACTS
-        pullSpec: brew.registry.redhat.io/rh-osbs/openshift-serverless-1-kn-cli-artifacts-rhel8:1.15.0
```
Comment (pierDipi) [inline config/client.yaml:8]: @dsimansk this is being deleted

Comment (pierDipi) [reply config/client.yaml:8]: /hold for now

Comment (pierDipi) [reply config/client.yaml:8]: Related to https://github.com/openshift-knative/hack/pull/459#discussion_r1862320348

Comment (dsimansk) [reply config/client.yaml:8]: IMO, fine to unhold and proceed. Per your other comment, It's not used per this config anyway.

File: pkg/konfluxgen/fbc-builder.yaml [hunk 2]
```diff
@@ -106,10 +84,20 @@ spec:
...
+  - default: ""
+    description: Path to a file with build arguments for buildah, see https://www.mankier.com/1/buildah-build#--build-arg-file
+    name: build-args-file
+    type: string
+  - default:
+    - linux/x86_64
+    description: List of platforms to build the container images on. The available
+      set of values is determined by the configuration of the multi-platform-controller.
+    name: build-platforms
+    type: array
```
Comment (dsimansk) [inline pkg/konfluxgen/fbc-builder.yaml:100]: @pierDipi let's merge this PR. It seems to be overlapping with multi-arch FBC changes.

File: config/client.yaml (hunk 1 above)
Comment (creydr) [reply config/client.yaml]: I think this is solved now (https://github.com/openshift-knative/hack/pull/479)
/unhold

**Review by creydr — APPROVED** [review]
/lgtm

**Review by creydr — APPROVED** [review]
/lgtm
/approve

**Pull request merged**